spyder/widgets/mixins.py

```diff
@@ -44,6 +44,11 @@
 
 class BaseEditMixin(object):
 
+    _PARAMETER_HIGHLIGHT_COLOR = '#DAA520'
+    _DEFAULT_TITLE_COLOR = '#2D62FF'
+    _CHAR_HIGHLIGHT_COLOR = 'red'
+    _DEFAULT_TEXT_COLOR = '#999999'

```

Comment:
I think some of these colors need to be different according to the current interface theme (dark/light).

For example, the current background color in the screenshots you uploaded is yellow but I think it should be the same (or perhaps a bit lighter) than the one used in the Editor.

Reply:
Yes, you are right, I will fix this!

Reply:
For now the colors used work for both themes, but perhaps we should indeed update the logic to use "better" colors. I think that deserves a separate PR

Reply:
https://github.com/spyder-ide/spyder/issues/9194